src/modules/LoggedIn/Project/ProjectForm.tsx

```diff
@@ -1,16 +1,27 @@
 import React, { ChangeEvent } from "react";
 import { Button, Modal } from "antd";
-import { useProjectSync, ProjectRequest } from "./useProjectSync";
-const ProjectForm = () => {
+
+import { useProjectSync, ProjectRequest, Project } from "./useProjectSync";
+
+type FormProps = {
+  visible: boolean;
+  toggleModal: () => void;
```

Comment:
nama props biasain pake prefix on, misal `onToggleModal`

```suggestion
  onToggleModal: () => void;
```